build/Makefile

```diff
@@ -512,12 +497,14 @@ endif
 #--------------------------------------------------------------------------------
 # Run unit tests
 #
+# Note: with the upgrade to ruby 2.3, rake needs the temporary PATH change to
```

Comment:
We do this routinely all over the place, and don't generally comment on how shell works. I think, when we're launching a shell command, it's reasonable to assume that the developer understands shell, or has the resources to look up what something does. If you want to comment, go ahead, but we do this sort of thing all over the place (thus my suggestion), and have never comment "Shell behaves like xxx yyy zzz" at those times.